AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -232,6 +232,8 @@ manager-magazin.de,whats-on-netflix.com#$#html.sp-message-open body { overflow-y
 !
 !
 !
+omni.se#%#//scriptlet('remove-class', 'sp-message-open', 'html')
+omni.se#$#div[id^="sp_message_container_"] { display: none !important; }
```

Comment:
Is something broken by `||cmp.omni.se/messagingNoTcfApi.js`?
Original report is about ad leftowers(I can't reproduce them)

Reply:
> Is something broken by `||cmp.omni.se/messagingNoTcfApi.js`?

Maybe not? I replace with it.

> Original report is about ad leftowers(I can't reproduce them)

I know. But, I think that the PR is necessary because there are no related rules in AdGuard Annoyance filter.